test/test_helper.rb

```diff
@@ -18,4 +18,8 @@ class ActiveSupport::TestCase
   fixtures :all
```

Comment:
Please ignore this file for now! I decided to work on this in a separate branch.